packages/app-form-builder/src/admin/plugins/editor/formFields/checkboxes.tsx

```diff
@@ -21,15 +21,16 @@ const plugin: FbBuilderFieldPlugin = {
                 name: this.name,
                 validation: [],
                 settings: {
-                    defaultValue: []
+                    defaultValue: [],
+                    otherOption: false
                 }
             };
         },
         renderSettings({ form }) {
             return (
                 <Grid>
                     <Cell span={12}>
-                        <OptionsList form={form} multiple />
+                        <OptionsList form={form} multiple otherOptionSwitch />
```

Comment:
```suggestion
                        <OptionsList form={form} multiple otherOption />
```

Let's rename to `otherOption`.